js/private/js_binary.bzl

```diff
@@ -155,6 +155,19 @@ _ATTRS = {
         """,
         default = True,
     ),
+    "preserve_symlinks_main": attr.bool(
+        doc = """When True, the --preserve-symlinks-main flag is passed to node.
+
+        This prevents node from following entry script out of runfiles and the sandbox which can happen for some entry
+        files such as `.mjs` where the fs node patches, which guard the runfiles and sandbox, are not used.
+
+        This flag was added in Node.js v10.2.0 (released 2018-05-23). If your node toolchain is configured to use a
+        Node.js version older than this you'll need to set this attribute to False.
```

Comment:
"If your node toolchain is configured to use a Node.js version older than this you'll need to set this attribute to False."

to 

"If your node toolchain is configured to use an older Node.js version this attribute must be set to False."

?

Reply:
You version is better but its merged already 🤷